### PR TITLE
protonvpn-gui: init at 2.1.1

### PR DESCRIPTION
**Pull request by offlinehacker**
###### Motivation for this change

Package `protonvpn-gui`, so i can have nice graphical interface and tray to connect to vpn.

###### Things done

This pull request does the following three things:

- Rename `protonvpn-cli-ng` to `protonvpn-cli`, as was renamed upstream https://github.com/protonvpn/protonvpn-cli-ng now points to https://github.com/ProtonVPN/linux-cli also add alias for old attribute name.
- Package `protonvpn-gui`

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

See also https://github.com/NixOS/nixpkgs/issues/93478 regarding the naming

### CODE REVIEW

**Comment by offlinehacker** [comment]
@GrahamcOfBorg build protonvpn-linux-gui

**Comment by offlinehacker** [comment]
CC @privateseabass

**Comment by privateseabass** [comment]
#93478
Personally, I'd just do this.
I don't know the reason people would still be using the original cli.

File: pkgs/applications/networking/protonvpn-linux-cli/default.nix [hunk 1]
```diff
@@ -17,6 +17,7 @@ python3Packages.buildPythonApplication rec {
       setuptools
       jinja2
       pythondialog
+      jinja2
```
Comment (jtcoolen) [inline pkgs/applications/networking/protonvpn-linux-cli/default.nix:20]: This line should be removed as jinja2 is added twice.

Comment (offlinehacker) [reply pkgs/applications/networking/protonvpn-linux-cli/default.nix]: Removed duplicate `jinja2`

**Comment by offlinehacker** [comment]
Updated according to: https://github.com/NixOS/nixpkgs/issues/93478

**Comment by offlinehacker** [comment]
I left the naming of the package as `protonvpn-linux-cli` and `protonvpn-linux-gui` regardless the attribute name is now `protonvpn-cli` and `protonvpn-gui`. These attributes can point to different derivations on different platforms, so in general having shorter names should be fine.

**Review by jtcoolen — APPROVED** [review]

**Pull request merged**